Cilicon/Config/GithubProvisionerConfig.swift

```
@@ -51,5 +55,9 @@ struct GithubProvisionerConfig: Decodable {
         self.url = try container.decodeIfPresent(URL.self, forKey: .url) ?? fallbackURL
         self.downloadLatest = try container.decodeIfPresent(Bool.self, forKey: .downloadLatest) ?? true
         self.workFolder = try container.decodeIfPresent(String.self, forKey: .workFolder)
+        self.livenessProbe = try container
+            .decodeIfPresent(LivenessProbeConfig.self, forKey: .livenessProbe) ?? LivenessProbeConfig(
+                command: "pgrep -fl /Users/admin/actions-runner/run.sh"
+            )
```

In case of `.livenessProbe` key is present but the decode fails, it will not fallback to value after `??`
Is what we want ?
Or do we need to use `try?`

---

I would rather fail the parsing of the config instead of silently falling back to default values.

---

yes, indeed